netket/nn/symmetric_linear.py

```diff
@@ -20,58 +20,18 @@
 import jax.numpy as jnp
 import numpy as np
 import jax
-from jax.nn.initializers import normal, zeros, lecun_normal, variance_scaling
+from jax.nn.initializers import zeros, lecun_normal
 
 from netket.utils import warn_deprecation
-from netket.nn.initializers import _complex_truncated_normal
 from netket.utils import HashableArray
 from netket.utils.types import Array, DType, PRNGKeyT, Shape, NNInitFunc
 from netket.utils.group import PermutationGroup
 from typing import Sequence
 from netket.graph import Graph, Lattice
 import warnings
 
-
-class _default_densesymm_initializer:
-    pass
-
-
-default_densesymm_initializer = _default_densesymm_initializer()
-"""
-Sentinel value to signal the default densesymm initializer.
-"""
-
-
-def get_default_densesymm_init(real_input_size, *, scale=1):
-    """
-    Returns a kernel init equivalent to lecun_normal but
-    whose scale depends on the number of physical degrees of freedom of the
-    layer.
-
-    This is the correct initialization to have in order to guarantee variance
-    1 of every output channel.
-    """
-
-    def init(key, shape, dtype):
-        shape = jax.core.as_named_shape(shape)
-
-        variance = jnp.array(scale / real_input_size, dtype=dtype)
-
-        if jnp.issubdtype(dtype, jnp.floating):
-            # constant is stddev of standard normal truncated to (-2, 2)
-            stddev = jnp.sqrt(variance) / 0.87962566103423978
-            return jax.random.truncated_normal(key, -2, 2, shape, dtype) * stddev
-        else:
-            stddev = jnp.sqrt(variance) / 0.95311164380491208
-            return _complex_truncated_normal(key, 2, shape, dtype) * stddev
-
-    return init
-
-
-def unit_normal_scaling(key, shape, dtype):
-    return jax.random.normal(key, shape, dtype) / jnp.sqrt(
-        jnp.prod(jnp.asarray(shape[1:]))
-    )
+# All layers defined here have kernels of shape [out_features, in_features, n_symm]
+default_gcnn_initializer = lecun_normal(in_axis=1, out_axis=0)
```

Comment:
I'm pretty sure `in_axis = (1,2)` here. And don't we have to specify `mode="fan_in"`?

Reply:
Yeah, that's confusing, `in_axis` is the axes that **only** appear in the input but not the output, so axis 2 is neither. And `lecun_normal` implies `fan_in`

Reply:
See the code [here](https://jax.readthedocs.io/en/latest/_modules/jax/_src/nn/initializers.html)

Reply:
Ahh I see, yes the code as written should work.